simplesamlphp_auth.admin.inc

```diff
@@ -45,13 +45,18 @@ function simplesamlphp_auth_settings() {
     '#default_value' => $config->get('authsource'),
     '#description' => t('The name of the source to use from /var/simplesamlphp/config/authsources.php'),
   );
+  $form['simplesamlphp_auth_grp_setup']['simplesamlphp_auth_login_link_display_name'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Federated Log In Link Display Name'),
+    '#default_value' => $config->get('link_display_name'),
```

Comment:
I believe I mistyped the configuration setting's name in my comment earlier so I'll notate where it needs to be updated. Here it should be:

`'#default_value' => $config->get('login_link_display_name'),`